new-pipeline/src/dashboards.js

```diff
@@ -1058,16 +1058,17 @@ function buildDictionaryURL(metric, channel, description) {
 
 function getDescriptionWithLink(metric, channel, description) {
   var metricUrl = buildDictionaryURL(metric, channel, description);
-  var descr = metric;
+  var descr;
+
   if (description && (description.length > 0)) {
-    descr = description;
+  descr = description;
```

Comment:
A small thing: This is missing indentation as this is inside a control block (`if` statement).